change/@ni-nimble-tokens-277e42ee-163c-4d3f-a31c-44a524b178a1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I looked around for any docs that needed updating. The only thing I found is a nit-pick: in `nimble-tokens/CONTRIBUTING.md` the styledictionary link points to `https://amzn.github.io/style-dictionary/#/` which still works but hasn't been updated for v4 yet and doesn't look as professional as `https://styledictionary.com/`. Mind updating it?